src/components/deck-gl-wrapper/mesh-layer/mesh-layer.ts

```diff
@@ -0,0 +1,193 @@
+// The file has been moved from deck.gl without changes to handle the fork of Tile3DLayer
```

Comment:
I wonder, if you make subclass of `Tile3DLayer`, do you still need this fork of `MeshLayer` here? It still yes, move it to `/src/layers/`

Reply:
Mesh Layer's code is removed